### ISSUE TITLE
About inplace operation error

### ISSUE THREAD

**Issue by Nyohohoho** [JiawangBian/SC-SfMLearner-Release#57]
I am trying to train the PoseNet (NeurIPS version) with additional loss functions.
When I run 'loss.backward()', PyTorch returned me some error information indicating to 'self.pose_pred' module.
This information is shown below:
> one of the variables needed for gradient computation has been modified by an inplace operation: [torch.cuda.FloatTensor [6, 256, 1, 1]] is at version 3; expected version 2 instead. Hint: the backtrace further above shows the operation that failed to compute its gradient. The variable in question was changed in there or anywhere later. Good luck!

I tried some solutions from the Internet, such as changing 'nn.ReLU(inplace=True)' to 'nn.ReLU(inplace=False)', but they did not work.

Could you please give me some advice about this?
I will be really thankful if you can give me your hands.
Thank you for the prompt attention to this matter.

**Comment by JiawangBian** [comment]
I guess that the new loss function is not correctly written. Could you check that again or give more information? Also, I suggest using the latest version, which is much better than NIPS version.

**Comment by Nyohohoho** [comment]
Thank you for your kind comments.
I am sorry to inform you late that I have solved this problem.
I really appreciate that you are sharing your code and helping people solve problems.
